### PR TITLE
news of the day

### PR DESCRIPTION
**Pull request by roman454647**

### CODE REVIEW

File: client/src/Routes.jsx [hunk 1]
```diff
@@ -22,7 +23,7 @@ export default class Routes extends Component {
             component={Home}
           />
         </Route>
-
+        <Route path='/newsoftheday' component={NewsOfTheDays} />
```
Comment (anna-kuzii) [inline client/src/Routes.jsx]: are u sure we need route? It is part of a home page

File: client/src/components/NewsOfTheDays/index.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import React, { Fragment } from 'react'
+import './style.scss'
+import { Link } from 'react-router-dom'
+
+export const NewsOfTheDays = (props) => {
+  const { image, tittle, text, photo_courtesy, full_news } = props
+
+  function showSeeMoreButton(text, full_news) {
+    return text.length <60 ? text
+      : <Fragment>{text.slice(0, 60)}<Link to={full_news} className='see-more'> ...</Link></Fragment>
```
Comment (anna-kuzii) [inline client/src/components/NewsOfTheDays/index.jsx]: why fragment?

File: client/src/components/NewsOfTheDays/index.jsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  }
+
+  return (
+    <div className='newsoftheday-wrapper'>
+      <div className='newsDivider'>
+        <hr />
+        <p>photo of the day</p>
+        <hr />
```
Comment (anna-kuzii) [inline client/src/components/NewsOfTheDays/index.jsx]: border

File: client/src/components/NewsOfTheDays/index.jsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  const { image, tittle, text, photo_courtesy, full_news } = props
+
+  function showSeeMoreButton(text, full_news) {
+    return text.length <60 ? text
+      : <Fragment>{text.slice(0, 60)}<Link to={full_news} className='see-more'> ...</Link></Fragment>
+  }
+
+  return (
+    <div className='newsoftheday-wrapper'>
+      <div className='newsDivider'>
```
Comment (anna-kuzii) [inline client/src/components/NewsOfTheDays/index.jsx]: what the convention? camelCase or -?

File: client/src/components/NewsOfTheDays/index.jsx [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+        <p>photo of the day</p>
+        <hr />
+      </div>
+      <div className='newsBlock'>
+        <img src={image} alt='background' />
+        <div className='photoOftheDay-triangle'>
+          <p>photo<span><br />of the<br /></span>day</p>
+        </div>
+        <div className='textOfNews'>
+          <h1 className='tittle'>{tittle}</h1>
```
Comment (anna-kuzii) [inline client/src/components/NewsOfTheDays/index.jsx]: TITLE!!!!!! two "T"!! NOT THREE

File: client/src/components/NewsOfTheDays/style.scss [hunk 6]
```diff
@@ -0,0 +1,136 @@
+@import '../styles/variables';
+@import '../styles/fonts';
```
Comment (anna-kuzii) [inline client/src/components/NewsOfTheDays/style.scss]: we discussed it today's morning. If u forgot -- ask Vitalii

File: client/src/Routes.jsx (hunk 1 above)
Comment (roman454647) [reply client/src/Routes.jsx]: removed

File: client/src/components/NewsOfTheDays/index.jsx (hunk 5 above)
Comment (roman454647) [reply client/src/components/NewsOfTheDays/index.jsx]: deleted

File: client/src/components/NewsOfTheDays/index.jsx (hunk 4 above)
Comment (roman454647) [reply client/src/components/NewsOfTheDays/index.jsx]: changed to camelcase

File: client/src/components/NewsOfTheDays/index.jsx (hunk 3 above)
Comment (roman454647) [reply client/src/components/NewsOfTheDays/index.jsx]: changed

File: client/src/components/NewsOfTheDays/index.jsx (hunk 2 above)
Comment (roman454647) [reply client/src/components/NewsOfTheDays/index.jsx]: changed

File: client/src/components/NewsOfTheDays/style.scss (hunk 6 above)
Comment (roman454647) [reply client/src/components/NewsOfTheDays/style.scss]: removed

File: client/src/components/NewsOfTheDays/index.jsx [hunk 7]
```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import './style.scss'
+import { Link } from 'react-router-dom'
+
+export const NewsOfTheDays = (props) => {
+  const { image, tittle, text, photo_courtesy, full_news } = props
```
Comment (anna-kuzii) [inline client/src/components/NewsOfTheDays/index.jsx]: roman -- title

Comment (roman454647) [reply client/src/components/NewsOfTheDays/index.jsx]: changed

File: client/src/components/NewsOfTheDays/index.jsx [hunk 8]
```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import './style.scss'
+import { Link } from 'react-router-dom'
+
+export const NewsOfTheDays = (props) => {
+  const { image, title, text, photo_courtesy, full_news } = props
+  function showSeeMoreButton(text, full_news) {
+    return text.length <100 ? text
+      : <p>{text.slice(0, 60)}<Link to={full_news} className='see-more'> ...</Link></p>
```
Comment (InnaKutsik) [inline client/src/components/NewsOfTheDays/index.jsx]: А натиснення на текст роутити не буде?

Comment (roman454647) [reply client/src/components/NewsOfTheDays/index.jsx]: ні,мені здається  коли на кнопку то зручніше

**Pull request merged**